### DISCUSSION TITLE
Es wird zensiert im https://www.ip-phone-forum.de Forum

### DISCUSSION THREAD

**Discussion by janfreetz** [Freetz-NG/freetz-ng#182] [Off-topic]
Hallo, habt Ihr das gelesen:

https://www.ip-phone-forum.de/threads/freetz-ng-fritzbox-st%C3%BCrzt-bei-verbindung-mit-vsftpd-ab.309351/

```

Hier war ein Link zu den Developern von Freetz-NG, der ist jetzt zensiert, gelöscht worden.
Warum wurde der Link gelöscht?
```

Das Admin, die Mods sind schon besonders.

**Comment by fda77** [comment]
Ich beteilige mich nicht mehr am IPPF, dort ist willkürliche Zensur an der Tagesordnung.
Vor kurzem wurde ich erst wieder gesperrt, weil ich das pöhse Wort "verarschen" benutzt hab.
Das ist mir zu blöd

**Comment by ghost** [comment]
Hier werden Issues oder Diskussionsbeiträge von fda77 mit sinnfreien Begründungen kommentiert und geschlossen.

**Reply by fda77** [reply]
.. weil irgend welche Schwachmaten ständig Fehler posten die durch ein **nicht** supportetes Addon verursacht werden: https://freetz-ng.github.io/freetz-ng/ADDONS Jetzt nerv hier nicht rum sondern beschwer dich beim Hersteller

**Reply by ghost** [reply]
Deine Behauptung entbehrt jeglicher Grundlage! Vielleicht solltest Du mal die Aschenbecher von Deinen Augen nehmen?

Nochmal extra für DICH zum NACHLESEN:
In meinem Beitrag, welchen Du geschlossen hast, steht MINIMAL Image und das heißt bei mir auch KEINE addons. Aber Du weißt ja alles besser! Sind wohl berechtigte Gründe, warum Du immer wieder beim IPPF gesperrt wirst. Und hier gibst Du nur Fehleinschätzungen von Dir.

Das Problem ist mit einer aktiven VDSL Verbindung verbunden. D.h. solange keine Verbindung besteht, funktioniert die ftp Verbindung. Wenn VDSL verbunden ist, stürzt die Box ab - auch mit der aktuellen TRUNK Version! Und nochmal für DICH: Git neu ausgecheckt, keine Addons, MINIMAL Konfiguration, nur vsftpd aktiviert.

Der Schwachmat bist eher Du selbst, weil Du offensichtlich nicht lesen kannst und meinst, alles aufgrund Deiner beschränkten Wahrnehmung FALSCH bewerten zu müssen.

**Comment by ghost** [comment]


**Reply by fda77** [reply]
Danke. Wenn man mehrmals beim Lügen ertappt wurde, wird es mit jedem mal schwerer. Ich hab aber wirklich keine Lust auf Addon-Talk

**Reply by Conan179** [reply]
> Während des Verbindens des ftp Clients (Total Commander) stürzt die Box ab.
> Wenn VDSL verbunden ist, stürzt die Box ab

Wie sich doch die ausagen mit der Zeit  ändern.

**Reply by ghost** [reply]
@fda77 
Deine Nase reicht schon bis zum Boden! Warum verbreitest Du hier so einen Unsinn? Ist doch offensichtlich, dass Du Dich irrst. Und trotzdem wiederholst Du diese Behauptung immer wieder und behauptest andere würden lügen. Jeder mit ausreichend Grips in der Birne würde anders reagieren. Ich erinnere nur an Python oder ncurses/ncursesw.  Dein Kommentar im IPPF war, dass Du von der Qualität Deiner Änderungen überzeugt bist, sich dies aber auch schnell mal ändern kann. Da hast Du mal  verhalten eingeräumt, dass Du Dich geirrt hast. Sonst weist Du das stets von Dir.`
`Ich möchte Dir raten, mit anderen stets höflich, kooperativ, auf Augenhöhe anstatt von oben herab zusammenzuarbeiten. Das scheint aber nicht Dein Stil zu sein.

@Conan179 
Ich habe halt weiter getestet und die entsprechende Beobachtung eingebaut. Ist das ein Problem? Beide Aussagen sind richtig.

Mehr als unpassende Kommentare oder Daumen hast Du nicht beizutragen?

**Reply by Conan179** [reply]
Im gegensatz zu dir, tu ich hier niemanden Beleidigen, nur weil ich nicht meinen Willen bekomme.
Ich teille kommplet die einschätzung von fda77.
Wen es dir nicht passt, nimm doch freetz.
EOD.

**Reply by kainers1** [reply]
> Wen es dir nicht passt, nimm doch freetz.

Oder noch besser einen eigenen Fork erstellen, dann kannst du über den bestimmen wie du möchtest.

Zum eigentlichen Problem: Logfiles wären sicherlich hilfreich. Oder habe ich sie übersehen?
Und ein Recovery dazwischen ist sinnvoll. Nur um sicherzustellen dass die Addons auch ganz weg sind. Diese Info habe ich bisher vermisst.

**Reply by Conan179** [reply]
@berndy2001 du hast nichts übersehen, das einzigte (absehen von beleidigungen) was hoch geladen worden ist, ist ne .config.

**Reply by ghost** [reply]
@berndy2001
Logdateien habe ich noch nicht. Ansonsten hatte ich bereits folgendes gemacht:
1. Minimal Image - nur mit vsftpd
2. Freetz Einstellungen gelöscht
3. Vergleichstest auf 2. Box - bei dieser ging es zunächst, weil offline
4. Boxen ausgetauscht 
5. Festplatten hin und her getauscht und Partitionsnamen geändert
6. Ergebnis dieser Tests war die Schlussfolgerung, dass die Abstürze nur auftreten, wenn VDSL Verbindung aktiv ist!
3. AVM Werkseinstellungen geladen
4. AVM Recovery
5. Wieder mit Minimal Image getestet
6. Heute nochmal mit aktuellem Git Trunk Minimal Image neu gebaut -> Abstürze treten weiterhin auf
7. Bftpd getestet -> funktioniert

**Reply by ghost** [reply]
GNU nano 5.3                                                     vsftpd.log
Sun Feb  7 22:27:48 2021 [pid 2] CONNECT: Client "192.168.1.39"
Sun Feb  7 22:27:48 2021 [pid 2] FTP response: Client "192.168.1.39", "220 (vsFTPd 3.0.3)"
Sun Feb  7 22:27:48 2021 [pid 2] FTP command: Client "192.168.1.39", "OPTS UTF8 ON"
Sun Feb  7 22:27:48 2021 [pid 2] FTP response: Client "192.168.1.39", "200 Always in UTF8 mode."

**Reply by Conan179** [reply]
> Minimal Image - nur mit vsftpd
> GNU nano 5.3

root@fritz:/var/mod/root# nano
-sh: nano: not found
root@fritz:/var/mod/root#

**Reply by ghost** [reply]
Ich habe nicht geschrieben, dass ich das Log mit Minimalimage erstellt habe. Das ist auch nicht erforderlich, weil das Fehlerbild bei allen Tests unabhängig von Git Version oder Minimalimage gleich ist.

**Reply by Conan179** [reply]
-.- Herzlichen Glückwunsch! Du hast es erfolgreich geschaft hier unsere Zeit zuverschwenden.
Du willst hier machts aber alles falsch was man nur machen kan, wen ein fehler angeblich beim mini image auftrit, dann macht mit dem weiter und macht nicht noch mehr ins Image wo noch mehr fehler prduzieren können.

@fda77 bitte schlies das hier, soviele lügen wie es hier gibt, bringt es nichts, weiter zu helfen!

**Reply by ghost** [reply]
@Conan179
Mehr als unpassende Kommentare oder Daumen hast Du **wieder** nicht beizutragen? Wie wäre damit: Du hast mit einer 7520/7530 getestet und dort tritt der Fehler nicht auf? Na dann erkläre mal, was ich **alles falsch** gemacht haben soll! Bin schon sehr gespannt. Logische Argumente sind nicht so Dein Ding.

**Comment by fda77** [comment]
Ich hab den ganzen Quatsch mal versteck. Es endelt leider immer so wenn forenuser/CommanderAdama hier was postet
